lib/ansible/plugins/action/win_reboot.py

```diff
@@ -81,8 +81,20 @@ def run(self, tmp=None, task_vars=None):
 
         result = super(ActionModule, self).run(tmp, task_vars)
 
-        # initiate reboot
-        (rc, stdout, stderr) = self._connection.exec_command("shutdown /r /t %d" % pre_reboot_delay_sec)
+        # Initiate reboot
+        (rc, stdout, stderr) = self._connection.exec_command('shutdown /r /t %d' % pre_reboot_delay_sec)
+
+        # Test for "A system shutdown has already been scheduled. (1190)" and handle it gracefully
+        if rc !=0 and '(1190)' in stderr:
```

Comment:
Given the dangers of localization (and that, at least in my testing, the RC matches the number in the text), I think I'd rather just check the explicit RC value 1190 here rather than parsing the output...